src/Ocelot/Cache/Middleware/OutputCacheMiddleware.cs

```diff
@@ -24,8 +24,8 @@ public OutputCacheMiddleware(RequestDelegate next,
     public async Task Invoke(HttpContext httpContext)
     {
         var downstreamRoute = httpContext.Items.DownstreamRoute();
-
-        if (!downstreamRoute.IsCached)
+        var options = downstreamRoute.CacheOptions;
+        if (!options.UseCache)
         {
             await _next.Invoke(httpContext);
             return;
```

Comment:
No need to await here since there’s no `try-catch` or similar handling. We should return the next `Task` immediately.